components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -325,6 +319,30 @@ - (void)setDrawerState:(MDCBottomDrawerState)drawerState {
   }
 }
 
+/**
+ The drawer height factor defines how much percentage of the screen space the drawer will take up
+ when displayed. The expected range is 0 - 1 (0% - 100%).
+
+ Default value is 0.5. If VoiceOver is enabled, or the mobile device is in landscape,
+ the default value becomes 1.0.
+ */
+- (CGFloat)initialDrawerFactor {
+  if ([self shouldPresentFullScreen]) {
+    return 1;
+  }
+  return _initialDrawerFactor;
+}
+
+- (void)setInitialDrawerFactor:(CGFloat)initialDrawerFactor {
+  if ([self shouldPresentFullScreen]) {
+    _initialDrawerFactor = 1;
```

Comment:
this if conditional is redundant, please remove it. You are also overriding it in the line after.